selfdrive/car/ford/interface.py

```diff
@@ -35,6 +35,16 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
     if candidate in CANFD_CAR:
       ret.safetyConfigs[-1].safetyParam |= Panda.FLAG_FORD_CANFD
 
+    # We have tested these values with CAN-FD Ford. This can be included for all vehicles after
+    # we have validated that CAN (Q3) Ford users also enjoy these changes.
```

Comment:
any Q3 test this yet? we can try it in our Bronco

Reply:
I've only known of a few of the Q3 vehicles decreasing their proportional and having had a better Long experience. I have not heard of anyone else with a Q3 trying any of the other settings. But as far as I know, most of them do not run Long or e2e.